src/main/java/codeu/controller/ProfileServlet.java

```diff
@@ -35,8 +36,18 @@ public void doGet(HttpServletRequest request, HttpServletResponse response)
       request.getRequestDispatcher("/WEB-INF/view/profile.jsp").forward(request, response);
       return;
     }
+    List<Message> authorMessages = new ArrayList<>();
+    for (UUID conversationId : user.getConversationIds()){
+      List<Message> messages = messageStore.getMessagesInConversation(conversationId);
+      for(Message message: messages){
+        if(message.getAuthorId().equals(user.getId())){
+          authorMessages.add(message);
+        }
+      }
+    }
```

Comment:
This algorithm works but I think that if we have a ton of messages it may be inefficient.